### DISCUSSION TITLE
Deploying your environment locally

### DISCUSSION THREAD

**Discussion by EY-glowglowglow** [EY-Data-Science-Program/2021-Better-Working-World-Data-Challenge#42] [3.0 VMs and data (Azure VM, local deployment, and Jupyter)]
Please refer [here ](https://github.com/EY-Data-Science-Program/2021-Better-Working-World-Data-Challenge/wiki/1c.-Getting-started:-Local-Environment) for instructions, and comment below if you get any errors with a screenshot of your error.

**Comment by AnishDixit** [comment]
I was following the azure setup instruction **1b. Getting started: Azure Environment**, when I received the following when deploying a data science environment. It showed that a VM Instance could not be created due to size limited exceeded for given region. I had put the region as Australia East only as recommended. Please advise on what to do.
![Screenshot (621)](https://user-images.githubusercontent.com/41044925/115502747-0902c200-a293-11eb-9d67-4e19c814e889.png)

**Reply by EY-glowglowglow** [reply]
This could be related to a quota limit already reached. As a first step, please [deallocate any existing VMs ](https://github.com/EY-Data-Science-Program/2021-Better-Working-World-Data-Challenge/wiki/3c.-Deleting-your-data-science-environment:-Azure) before deploying a new one.
If that does not work, please try one of the following: 
•	If you are using Azure for Students, you will not be able to increase the quota. You would need to deallocate existing VMs to deploy new ones. 
•	If you're using a different subscription (not Azure for Students) you can increase the quota following instructions [here](https://docs.microsoft.com/en-us/azure/azure-portal/supportability/per-vm-quota-requests). Please note that there may be costs associated with this.

**Reply by lovegarg97** [reply]
I am facing the same issue. I dellocated existing VMs, still it was not working.

**Reply by mohammadshaheer4** [reply]
I'm facing a similar error. I don't have any existing VM's but still, I'm facing the error. Is there a workaround?

**Reply by EY-glowglowglow** [reply]
Hi there, @lovegarg97  @mohammadshaheer4  can you please try Australia Southeast instead of Australia East? Thank you!

**Comment by data-adrian** [comment]
Hello, 

I have followed the instructions and setup the env, but when I run dc = Datacube(), I get the following error. Is there an existing solution for this?

IndexSetupError: 

No DB schema exists. Have you run init?
	datacube system init

<img width="1067" alt="Screen Shot 2021-05-15 at 11 50 02 am" src="https://user-images.githubusercontent.com/49013526/118344852-e2733680-b573-11eb-8ded-35c352d46a27.png">

**Reply by ymisra** [reply]
I am facing the exact same problem

**Reply by EY-glowglowglow** [reply]
@alexgleith can you have a look at this please?

**Reply by alexgleith** [reply]
Is this from launching your instance from a template, or a self-managed instance?

If it's from a template, I recommend destroying your resource group and starting again.

If it's manual, please follow these steps: https://github.com/EY-Data-Science-Program/2021-Better-Working-World-Data-Challenge/wiki/1c.-Getting-started:-Local-Environment#step-3-populate-the-data-cube
